src/main/scala/org/openeo/opensearch/backends/CreodiasClient.scala

```diff
@@ -48,6 +48,7 @@ object CreodiasClient extends OpenSearchClient {
       .param("status", "0|34|37")
       .param("dataset", "ESA-DATASET")
       .params(attributeValues.mapValues(_.toString).filterKeys(!Seq( "eo:cloud_cover", "provider:backend", "orbitDirection", "sat:orbit_state").contains(_)).toSeq)
+      .timeout(connTimeoutMs = 10000, readTimeoutMs = 40000)
```

Comment:
This is already covered by the change in OpenSearchClient.

Reply:
fixed